examples/custom_projector/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/etf1/kafka-transformer/pkg/transformer/kafka"
+	confluent "gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
+)
+
+// Start a redis with docker : docker run -p 6379:6379 --rm -d redis
```

Comment:
Maybe you can add a redis instance in the docker-compose file ?

Reply:
The docker-compose is for the kafka-transformer project. kafka-trasnformer is not using a redis projector. That is why i didn't add a redis server in the Docker-compose.yml.

The examples directory is a sub gomodule, totally independent. But if you want i can add a docker-compose.yml in this sub project ?

Reply:
OK for me to add a docker-compose file in this sub projet.

Reply:
Done